.ci.yaml

```diff
@@ -61,8 +61,8 @@ platform_properties:
           {"name":"osx_sdk_13a233_13_15","path":"osx_sdk"},
           {"name":"builder_mac_engine","path":"builder"},
           {"name":"openjdk","path":"java"},
-          {"name":"xcode_runtime_ios-13-0","path":"xcode_runtime_ios-13-0"},
-          {"name":"xcode_runtime_ios-15-0","path":"xcode_runtime_ios-15-0"}
+          {"name":"xcode_runtime_ios_13_0","path":"xcode_runtime_ios-13-0"},
```

Comment:
This comment should be updated then?
https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/osx_sdk/api.py#96

I'm assuming the cache names don't matter since I previously made up `osx_sdk_13a233_13_15`

Reply:
I'll send the recipes CL, and hopefully no person has to worry about whether your cache has a dash or not...